.github/workflows/main.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         id: artifacts-directory
         run: |
           VERSION=$(git tag --points-at HEAD)
-          echo "::set-output name=path::content/leaflet/${VERSION:-master}"
+          echo "::set-output name=path::content/leaflet/${VERSION:-main}"
```

Comment:
I've also updated the path where previously the `master` version would be uploaded, it seems that all these references have been updated. It might however be necessary for this directory to be removed from the Amazon bucket and set up redirects (if possible).

If this is too much effort (or breaks links out in the wild) we might consider leaving this set to `master` and stick with just the branch rename for now.

Reply:
Let's keep existing links working. New builds will go to updated path and that's good enough IMO.